README.md

```diff
@@ -68,6 +68,10 @@ node_modules/bootstrap/dist/js/bootstrap.js in the projects->architect->build->s
 node_modules/bootstrap/dist/js/bootstrap.js in the projects->architect->build->scripts array,
 ```
 
+<<<<<<< HEAD
```

Comment:
Take a merge from main branch and resolve this conflict then you can push the code.